### ISSUE TITLE
About training with batchsize==1 in semantic segmentation task

### ISSUE THREAD

**Issue by Solacex** [NVIDIA/apex#484]
Does apex support training with batchsize==1 in semantic segmentation task?

**Comment by ptrblck** [comment]
`apex` does not introduce a requirement for a minimal batch size, so if the code is running fine in plain PyTorch, it should also run using `apex`.

Do you see any errors?

**Comment by Solacex** [comment]
> `apex` does not introduce a requirement for a minimal batch size, so if the code is running fine in plain PyTorch, it should also run using `apex`.
> 
> Do you see any errors?

Yes, I try to train a model(DeeplabV2 with resnet101) with fixed batch norm and batchsize==1, apex can not produce the same result as model trained without apex.
I notice that the initial loss value of the two(with/without apex) are very similar ( difference in 0.005), but the model with apex can not converge finally.
